MapboxNavigation.xcodeproj/xcshareddata/xcschemes/Example.xcscheme

```diff
@@ -1,99 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Scheme
```

Comment:
Remember to share any relevant schemes as part of the new Xcode project?

Reply:
Done